travis.txt

```diff
@@ -2,6 +2,8 @@
 # versions.
 mock==1.3.0
 Jinja2>=2.7.1
+dj-database-url==0.5.0
```

Comment:
This just provides a convenient way of parsing the `DATABASE_URL` environment variable. I felt it was fairly harmless to add a dependency here, since these are after all only the CI requirements.

Reply:
I feel like I've implemented this myself with `urlparse` so many times. TIL!